core/domain/exp_services_test.py

```diff
@@ -4887,6 +4887,284 @@ def test_get_composite_change_list(self):
         self.assertEqual(
             composite_change_list_dict_expected, composite_change_list_dict)
 
+    def test_reverts_exp_to_safe_state_when_content_model_is_missing(self):
```

Comment:
Can you please add a test where the exploration is reverted for more than one version?

Reply:
Done.